src/devices/Is31fl3731/Bonnet16x8.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.Display
+{
+    /// <summary>
+    /// Represents 16x8 matrix, driven by an IS31FL3731 LED chip.
+    /// </summary>
+    // Product: https://www.adafruit.com/product/4122
+    public class Bonnet16x8 : Is31fl3731
+    {
+        /// <summary>
+        /// Initialize LED driver.
+        /// </summary>
+        /// <param name="i2cDevice">The <see cref="System.Device.I2c.I2cDevice"/> to create with.</param>
+        public Bonnet16x8(I2cDevice i2cDevice)
+        : base(i2cDevice, 16, 8)
+        {
+        }
+
+        /// <summary>
+        /// Default I2C address for device.
+        /// </summary>
+        public static new readonly int DefaultI2cAddress = 0x74;
```

Comment:
Because this code is hiding an inherited member, which requires `new`.

```csharp
public static const int DefaultI2cAddress = 0x74;
```

Produces the following error:

> Bonnet16x8.DefaultI2cAddress' hides inherited member 'Is31fl3731.DefaultI2cAddress'. Use the new keyword if hiding was intended. [Is31Fl3731]csharp(CS0108)